From 329423806e14df4d1f5fefe029cb713433793a0a Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 10 Feb 2011 13:08:08 +0000
Subject: [PATCH] Use --search by default in the desktop file

So that you don't need to launch gnote twice to get
anything to appear.
---
 data/gnote.desktop.in.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/gnote.desktop.in.in b/data/gnote.desktop.in.in
index 890fc89..24966a5 100644
--- a/data/gnote.desktop.in.in
+++ b/data/gnote.desktop.in.in
@@ -3,7 +3,7 @@
 _Name=Gnote
 _Comment=Take notes, link ideas, and stay organized
 _GenericName=Note-taker
-Exec=gnote
+Exec=gnote --search
 Icon=gnote
 StartupNotify=false
 Terminal=false
-- 
1.7.4

